### PR TITLE
79 coco detection splitmixin

### LINKED ISSUE #79: Update COCO dataset and COCODetection to detr with  SplitMix class
Update COCO detection with SplitMix class.

### PR DESCRIPTION
**Pull request by Johansmm**
New `alodataset` : `CocoExtendDetectionDataset`. New version of `CocoDetectionDataset` with the following new features:  
 * SplitMixin mode compatibility
 * Include stuff categories from stuff_annotations.json
 * Return multiples labels found in annotations (supercategory, isthing, iscrowd).

Also, update of `CocoDetectionDataset` to incorporate *return multiples labels* feat.

Closes #79 #97

### CODE REVIEW

File: alodataset/coco_extend_detection_dataset.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+import torch
+import numpy as np
+from pycocotools.coco import COCO
+from collections import defaultdict
+
+from alodataset import BaseDataset, CocoDetectionDataset, Split, SplitMixin
+from aloscene import Frame
+
+
+class CocoExtendDetectionDataset(CocoDetectionDataset, SplitMixin):
```
[the remark this answers is not in the document]
Comment (Johansmm) [reply alodataset/coco_extend_detection_dataset.py]: Done

**Review by thibo73800 — APPROVED** [review]
